src/AggregateFunctions/AggregateFunctionWindowFunnel.h

```diff
@@ -150,7 +150,7 @@ class AggregateFunctionWindowFunnel final
     /// Loop through the entire events_list, update the event timestamp value
     /// The level path must be 1---2---3---...---check_events_size, find the max event level that satisfied the path in the sliding window.
     /// If found, returns the max event level, else return 0.
-    /// The Algorithm complexity is O(n).
+    /// The algorithm works in O(n * log(n)) time.
```

Comment:
OK, but maybe it would be better to mention that this asymptotic is only due to sorting. The described in the comment algorithm is works blatantly in O(n), so the reader can be confused. Something like this:
```suggestion
    /// The algorithm works in O(n) time, but the overall function works in O(n * log(n)) due to sorting.
```